src/lib/components/workflow/workflow-count.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import type { WorkflowStatus as Status } from '$lib/types/workflows';
+  import WorkflowStatus from '../workflow-status.svelte';
+
+  export let status: Status;
+  export let count: number = Math.round(Math.random() * 10000);
```

Comment:
Seems odd to have a default value for the count, is this to avoid an empty box when the counts are loading? Should we use a loading indicator instead?

Reply:
Oh I see this is just for testing purposes. Can remove the default value once we get the API 👍

Reply:
Maybe we can add a `TODO` here?